process-migration-service/frontend/src/component/tabMigrationPlan/wizardAddPlan/PageDefinition.js

```diff
@@ -1,41 +1,90 @@
 import React, { Component } from "react";
-import axios from "axios";
 
 import { Button } from "patternfly-react";
+import { ALERT_TYPE_ERROR } from "patternfly-react/dist/js/components/Alert/AlertConstants";
 
 import PageDefinitionSearchTable from "./PageDefinitionSearchTable";
 import { BACKEND_URL } from "../../common/PimConstants";
+import Notification from "../../Notification";
 
 export default class PageDefinition extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      errorMsg: ""
+    };
   }
 
   copySourceToTarget = () => {
     this.props.onChangeTargetContainerId(this.props.sourceContainerId);
     this.props.onChangeTargetProcessId(this.props.sourceProcessId);
   };
 
-  getDefinitions = () => {
-    const servicesUrl = BACKEND_URL + "/kieserver/definitions";
-    axios
-      .get(servicesUrl, {
-        params: {
-          sourceProcessId: this.props.sourceProcessId,
-          sourceContainerId: this.props.sourceContainerId,
-          targetProcessId: this.props.targetProcessId,
-          targetContainerId: this.props.targetContainerId,
-          kieServerId: this.props.kieServerIds
+  getDefinition = (containerId, processId, callbackFn) => {
+    const defURL = `${BACKEND_URL}/kieserver/${
+      this.props.kieServerId
+    }/definitions/${containerId}/${processId}`;
+    fetch(defURL, {
```

Comment:
again using fetch vs. axios.get() ... if there's not any reason to mix the two, I'd stick with one way.